TwitterChallenge.html

```diff
@@ -0,0 +1,433 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="StyleTwitter.css" />
+    <title>Twitter Clone</title>
+</head>
+<body>
```

Comment:
I like the way you have structured the HTML - looks very organised!
Could write some comments to section each section, for example, 
<!--Left bar starts-->
Code
<!--Left bar ends-->

This is so if you need to change anything in your HTML  - you can quickly find the section you want to change.